about.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="about.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>About Page</title>
+  </head>
+  <body>
+    <div class="background1">
+      <section class="navbar">
+        <div class="mobile-nav">
+          <a href="#">
+            <img
+              id="mobile-cta"
+              class="mobile-menu"
+              src="./images/hamburguer-menu.svg"
+              alt="menu-icon"
+            />
+          </a>
+        </div>
+
+        <nav>
+          <img
+            id="mobile-exit"
+            class="mobile-exit"
+            src="./images/close-icon.svg"
+            alt="close-navigation"
+          />
+          <a href="index.html" class="logo-tag">
+            <img
+              id="logo"
+              class="logo"
+              src="./images/logo.png"
+              alt="logo-image"
+            />
+          </a>
+          <ul class="primary-nav">
+            <li>
+              <a id="menu-about" class="menu-link about-link" href="about.html"
+                >About</a
+              >
+            </li>
+            <li>
+              <a id="menu-program" class="menu-link" href="index.html"
+                >Program</a
+              >
+            </li>
+            <li>
+              <a id="menu-join" class="menu-link" href="index.html">Join</a>
+            </li>
+            <li>
+              <a id="menu-sponsor" class="menu-link" href="#sponsor">Sponsor</a>
+            </li>
+            <li>
+              <a id="menu-news" class="menu-link" href="#news">News</a>
+            </li>
+          </ul>
+          <a href="#" class="course-button">Log Course</a>
+        </nav>
+      </section>
```

Comment:
- [ ] The navbar is of two parts, the black and the white parts. Your project is missing the black part of the navbar. Kindly add that part with all its elements. Look at the two designs below.

### Your Design

![yournavbar](https://user-images.githubusercontent.com/26597432/196833335-c9aecf6f-4542-4845-adec-685c3c52194c.png)


### Guiding Template

![homepagenavbar](https://user-images.githubusercontent.com/26597432/196833392-4bd118e1-4172-494a-a61c-539cb6660aae.png)


- [OPTIONAL] I would suggest you increase the font-size of all your menu items. Presently, it look too small and that is not good enough.

![menu-items-size](https://user-images.githubusercontent.com/26597432/196834403-9bc92ecb-ac32-4b58-804e-ad57e4a0c72a.png)